### PR TITLE
Contributing file

### PR DESCRIPTION
**Pull request by azerupi**
This adds a "how to contribute" guide per #270 

Let me know if you would change or add anything :)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+Issues on the issue tracker are categorized with the following labels:
+
+- **A**-prefixed labels state which area of the project an issue relates to.
+- **E**-prefixed labels show an estimate of the experience necessary to fix the issue.
+- **M**-prefixed labels are meta-issues used for questions, discussions, or tracking issues
+- **S**-prefixed labels show the status of the issue
+- **T**-prefixed labels show the type of issue
+
+### Building mdBook
+mdBook builds on stable Rust, if you want to build mdBook from source, here are the steps to follow:
```
Comment (steveklabnik) [inline CONTRIBUTING.md]: i'd add a newline above this line

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+1. Navigate to the directory of your choice
+0. Clone this repository with git.
+   ```
```
Comment (steveklabnik) [inline CONTRIBUTING.md]: same here

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+   git clone https://github.com/azerupi/mdBook.git
+   ```
+0. Navigate into the newly created `mdBook` directory
+0. Run `cargo build`
+
+The resulting binary can be found in `mdBook/target/debug/` under the name `mdBook` or `mdBook.exe`.
+
+
+### Making changes to the style
+mdBook doesn't use CSS directly but uses [Stylus](http://stylus-lang.com/), a CSS-preprocessor which compiles to CSS.
```
Comment (steveklabnik) [inline CONTRIBUTING.md]: and here

**Comment by azerupi** [comment]
Thanks for the feedback!
I added the newlines and wrapped the lines at 120.

**Comment by azerupi** [comment]
Good, let's merge it then :)

**Pull request merged**